tests/handler_tests/test_tripadvisor_handler.py

```diff
@@ -0,0 +1,157 @@
+from mindsdb.integrations.handlers.tripadvisor_handler.tripadvisor_handler import (
+    TripAdvisorHandler,
+)
+
+from mindsdb.integrations.handlers.tripadvisor_handler.tripadvisor_table import (
+    SearchLocationTable,
+    LocationDetailsTable,
+)
+
+from mindsdb_sql.parser import ast
+from mindsdb_sql.parser.ast.select.star import Star
+
+from unittest.mock import Mock
+
+import pandas as pd
+import unittest
+
+
+class SearchLocationTableTest(unittest.TestCase):
+    def test_get_columns_returns_all_columns(self):
+        api_handler = Mock(TripAdvisorHandler)
+        trades_table = SearchLocationTable(api_handler)
+        # Order matters.
+        expected_columns = [
+            "location_id",
+            "name",
+            "distance",
+            "rating",
+            "bearing",
+            "street1",
+            "street2",
+            "city",
+            "state",
+            "country",
+            "postalcode",
+            "address_string",
+            "phone",
+            "latitude",
+            "longitude",
+        ]
+        self.assertListEqual(trades_table.get_columns(), expected_columns)
+
+    def test_select_returns_some_columns(self):
+        api_handler = Mock(TripAdvisorHandler)
+        api_handler.call_tripadvisor_searchlocation_api.return_value = pd.DataFrame(
+            [
+                [
+                    "186338",  # locationId
+                    "London",  # name
+                    "United Kingdom",  # country
+                    "London England",  # address_string
+                ]
+            ]
+        )
+        tripadvisor_table = SearchLocationTable(api_handler)
+
+        select_all = ast.Select(
+            targets=[Star()],
+            from_table="searchLocationTable",
+            where='searchLocationTable.searchQuery = "London"',
+        )
+
+        all_location_data = tripadvisor_table.select(select_all)
+        first_data = all_location_data.iloc[0]
+
+        self.assertEqual(all_location_data.shape[1], 4)
+        self.assertEqual(first_data["locationId"], "186338")
+        self.assertEqual(first_data["name"], "London")
+        self.assertEqual(first_data["country"], "United Kingdom")
+        self.assertEqual(first_data["address_string"], "London England")
+
+
+class LocationDetailsTableTest(unittest.TestCase):
```

Comment:
Let's move all `tests` to the tests directory.

Reply:
Alright got it, will do!

Reply:
Wait, isn't this test script alreadyy in _tests_ directory? 

[**tests**/handler_tests/test_tripadvisor_handler.py]

Reply:
Apologies, my mistake. It is. Let's move the other tests in to this directory as well.

Reply:
Oh got it!